cmd/kops/update_cluster.go

```diff
@@ -65,6 +65,7 @@ type UpdateClusterOptions struct {
 	SSHPublicKey    string
 	MaxTaskDuration time.Duration
 	CreateKubecfg   bool
+	NodeUpSource    string
```

Comment:
I see now. You can set the same value using the `NODEUP_URL` env var. OK, will remove the 2nd commit then.